src/main/java/com/skynest/uitesting/config/properties/PropertiesReader.java

```diff
@@ -0,0 +1,34 @@
+package com.skynest.uitesting.config.properties;
+
+import org.apache.commons.configuration.CompositeConfiguration;
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.commons.configuration.PropertiesConfiguration;
+
+public class PropertiesReader {
+    private static PropertiesReader instance;
+    private static CompositeConfiguration configuration;
+
+    private PropertiesReader() {
+        configuration = new CompositeConfiguration();
+        try {
+            configuration.addConfiguration(new PropertiesConfiguration("config.properties"));
+        } catch (ConfigurationException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static PropertiesReader getInstance() {
```

Comment:
My sonarlint plugin complained about this method and the following changes seem to have fixed it:
```
    public static synchronized PropertiesReader getInstance() {
        if (instance == null) {
            instance = new PropertiesReader();
        }
        return instance;
    }
```
I strongly recommend using this plugin - more often than not, it has useful suggestions.